<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Client</class>
 <widget class="QWidget" name="Client">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>792</width>
    <height>542</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>792</width>
    <height>542</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>792</width>
    <height>542</height>
   </size>
  </property>
  <property name="styleSheet">
   <string notr="true">
    QProgressBar::chunk {
      background-color: #00cc00;
      width: 1px;
    }
    QProgressBar {
      border-radius: 0px;
      text-align: center;
    }
   </string>
  </property>
  <widget class="CPlotter" name="Plotter">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>791</width>
     <height>301</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>771</width>
     <height>301</height>
    </size>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>300</y>
     <width>791</width>
     <height>241</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QTabWidget::pane {border: 1px solid #C2C7CB;}
QTabWidget::tab-bar {left: 5px;}
QTabBar::tab {background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #E1E1E1, stop: 0.4 #DDDDDD, stop: 0.5 #D8D8D8, stop: 1.0 #D3D3D3); border: 2px solid #C4C4C3; border-top-color: #C2C7CB; border-bottom-left-radius: 4px; border-bottom-right-radius: 4px; min-width: 8ex; padding: 2px;}
QTabBar::tab:selected, QTabBar::tab:hover {background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #fafafa, stop: 0.4 #f4f4f4, stop: 0.5 #e7e7e7, stop: 1.0 #fafafa);}
QTabBar::tab:selected {border-color: #9B9B9B; border-top-color: #C2C7CB;}
QTabBar::tab:!selected {margin-bottom: 2px;}</string>
   </property>
   <property name="tabPosition">
    <enum>QTabWidget::South</enum>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Rounded</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="Main">
    <attribute name="title">
     <string>Main</string>
    </attribute>
    <widget class="QPushButton" name="connect">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>161</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Connect</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Device</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>60</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Bandwidth</string>
     </property>
    </widget>
    <widget class="QPushButton" name="EnableTX">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>10</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>TX</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>110</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Device</string>
     </property>
    </widget>
    <widget class="QSlider" name="Range">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>80</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>5</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
     <property name="pageStep">
      <number>1</number>
     </property>
     <property name="value">
      <number>4</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QComboBox" name="DeviceTX">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>130</y>
       <width>231</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QSlider" name="GainTX">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>180</y>
       <width>231</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QComboBox" name="FilterTX">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>80</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QComboBox" name="ModulationTX">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>80</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QSlider" name="Offset">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>130</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>40</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
     <property name="pageStep">
      <number>2</number>
     </property>
     <property name="value">
      <number>16</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QProgressBar" name="LevelRX">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>231</width>
       <height>11</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="value">
      <number>24</number>
     </property>
     <property name="textVisible">
      <bool>false</bool>
     </property>
     <property name="format">
      <string>%p%</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>60</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>AGC</string>
     </property>
    </widget>
    <widget class="QSlider" name="GainRX">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>231</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="CopyTXRX">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>31</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>&lt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="EnableFFT">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>10</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>FFT</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="Indicator" name="IndicatorFFT">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>10</y>
       <width>181</width>
       <height>41</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
    </widget>
    <widget class="QComboBox" name="FilterRX">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>80</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Modulation</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>110</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Offset</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>60</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Filter</string>
     </property>
    </widget>
    <widget class="QComboBox" name="DeviceRX">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>231</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>60</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Filter</string>
     </property>
    </widget>
    <widget class="QComboBox" name="AutoGainTX">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>80</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QComboBox" name="Address">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>180</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QProgressBar" name="LevelTX">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>160</y>
       <width>231</width>
       <height>11</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="value">
      <number>24</number>
     </property>
     <property name="textVisible">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="Indicator" name="IndicatorTX">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>10</y>
       <width>181</width>
       <height>41</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="CopyRXTX">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>11</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>&gt;</string>
     </property>
    </widget>
    <widget class="QComboBox" name="Width">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>80</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QSpinBox" name="Correction">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>130</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="minimum">
      <number>-100</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>25</number>
     </property>
    </widget>
    <widget class="Indicator" name="IndicatorRX">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>10</y>
       <width>181</width>
       <height>41</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>60</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>AGC</string>
     </property>
    </widget>
    <widget class="QComboBox" name="AutoGainRX">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>80</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>60</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Range</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>60</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Modulation</string>
     </property>
    </widget>
    <widget class="QPushButton" name="EnableRX">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>RX</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QComboBox" name="ModulationRX">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>160</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>IP address</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_25">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>110</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Freq. corr. (ppm)</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="SettingsRX">
    <attribute name="title">
     <string>RX settings</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="SettingsTX">
    <attribute name="title">
     <string>TX settings</string>
    </attribute>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Indicator</class>
   <extends>QFrame</extends>
   <header>Indicator.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CPlotter</class>
   <extends>QFrame</extends>
   <header>plotter.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
